modules/axhal/src/arch/x86_64/trap.rs

```diff
@@ -78,6 +78,7 @@ fn vec_to_str(vec: u64) -> &'static str {
 fn err_code_to_flags(err_code: u64) -> Result<MappingFlags, u64> {
     let code = PageFaultErrorCode::from_bits_truncate(err_code);
     let reserved_bits = (PageFaultErrorCode::CAUSED_BY_WRITE
+        | PageFaultErrorCode::PROTECTION_VIOLATION
```

Comment:
@mingzi47 Sorry to bother you, I was wondering if this change is necessary? Because I'm porting these to `axcpu`.